$MirOS$

	Replace with version which actually does what they intended to.

--- src/runtime/Integer/alloca.c.orig	Mon Oct 11 11:50:01 1999
+++ src/runtime/Integer/alloca.c	Sat Apr 18 20:32:09 2009
@@ -1,8 +1,11 @@
 #include <stdio.h>
+#include <stdlib.h>
 
+__RCSID("$MirOS$");
 
-void *alloca (unsigned size)
+void *
+alloca(size_t size)
 {
-  fprintf(stderr,"Falilure alloca %d called:-(\n",size);
-  abort(-1);
+	fprintf(stderr, "Failure alloca %zu called :-( \n", size);
+	abort();
 }
